packages/amplify_auth_cognito/android/build.gradle

```diff
@@ -51,7 +51,7 @@ android {
 
```

Comment:
Why is the minSdkVersion not changed here, but is changed in analytics?

Reply:
it's because with the latest upgrade, the analytics plugin is now over 64K methods, which is over the limit of a single android application unless multiDex is enabled or minsdk version is changed to 21 https://developer.android.com/studio/build/multidex. I and Jameson thought it's ok to increase the minsdk version for our flutter library to 21 instead of dealing with enabling multidex support.